Documentation/etcd-mixin/test.yaml

```diff
@@ -71,18 +71,16 @@ tests:
       - series: 'up{job="etcd",instance="10.10.10.1"}'
         values: '1 1 1 1 1 1 1 1 0 0 0 0 0 0 0 0'
       - series: 'etcd_network_peer_sent_failures_total{To="member-1",job="etcd",endpoint="test"}'
-        values: '0 0 1 2 3 4 5 6 7 8 9 10'
+        values: '0 0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18'
     alert_rule_test:
-      - eval_time: 4m
-        alertname: etcdMembersDown
-      - eval_time: 6m
+      - eval_time: 13m
         alertname: etcdMembersDown
         exp_alerts:
           - exp_labels:
               job: etcd
               severity: critical
             exp_annotations:
-              message: 'etcd cluster "etcd": members are down (1).'
+              message: 'etcd cluster "etcd": members are down (2).'
```

Comment:
I don't actually understand what caused the member ID to change in this scenario; I don't see how the etcd_network_peer_sent_failures_total selector correlates to any given `instance` in the series setup.

Reply:
I am fuzzy here, but the alert's existing `count without (To) (` feels like it should maybe be `count by (To)`, i.e. "talking to the members that are up and serving metrics, here's who they are complaining about.

Reply:
Need to think about that some more... not sure if I want to change it with this PR. Fixing the padding in the test did fix the message expectation, still not entirely clear why